server/config/config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "development": {
+    "username": "root",
+    "password": null,
+    "database": "mentel_db",
+    "host": "127.0.0.1",
+    "dialect": "mysql",
+    "operatorsAliases": 0
+  },
+  "test": {
+    "username": "root",
+    "password": null,
+    "database": "database_test",
+    "host": "127.0.0.1",
+    "dialect": "mysql",
+    "operatorsAliases": false
+  },
+  "production": {
+    "username": "root",
+    "password": null,
+    "database": "database_production",
+    "host": "127.0.0.1",
+    "dialect": "mysql",
+    "operatorsAliases": false
```

Comment:
Use `dotenv` to store sensitive data that can be used to exploit your app
check `dotenv` np package